packages/builder/src/components/common/LinkedRecordSelector.svelte

```diff
@@ -0,0 +1,74 @@
+<script>
```

Comment:
In general for this - I know this is WIP.... but i think we will want a typeahead here. Which means we will need the "Primary Field" concept... like Airtable.

the `all_<modelId>` view should then be keyed by this